src/core/geometry/qgspolyhedralsurface.h

```diff
@@ -0,0 +1,340 @@
+/***************************************************************************
+                         qgspolyhedralsurface.h
+                         -------------------
+    begin                : August 2024
+    copyright            : (C) 2024 by Jean Felder
+    email                : jean dot felder at oslandia dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSPOLYHEDRALSURFACE_H
+#define QGSPOLYHEDRALSURFACE_H
+
+#include "qgis_core.h"
+#include "qgis_sip.h"
+#include "qgssurface.h"
+#include "qgspolygon.h"
+#include "qgsmultipolygon.h"
+
+
+/**
+ * \ingroup core
+ * \class QgsPolyhedralSurface
+ * \brief polyhedral surface geometry type
```

Comment:
Can you add more detail here? There should be sufficient information included so that it's immediately clear what a polyhedral surface type is and how/why it differs from regular polygon types.

(Also make sure the docs are properly capitalised)

Reply:
Fixed.